### PR TITLE
fix(github): dependabot remove ignore

### PR DESCRIPTION
**Pull request by johannbg**
That's rather unhelpful error msg dependabot...
Removed the ignore line since I suspect it's the name matching that is failing and dependabot files PR for changes so we can just nack those that we dont want to update.

### CODE REVIEW

File: .github/dependabot.yml [hunk 1]
```diff
@@ -4,11 +4,11 @@ version: 2
   updates:
```
Comment (LaszloGombos) [inline .github/dependabot.yml:4]: Why are "updates" intended with 2 white spaces "  " ? Is that significant.

The example github generates does not have those indentation.

**Review by LaszloGombos — COMMENTED** [review]
I suspect dependabot will also run in our personal forks. I hope we find a way to disable or at least make it not very annoying. Not meaning to stop this work, just a heads-up on what to expect to come.

File: .github/dependabot.yml (hunk 1 above)
Comment (aafeijoo-suse) [reply .github/dependabot.yml:4]: Right, we missed that in #2016. Check https://jsonformatter.org/yaml-validator

**Comment by LaszloGombos** [comment]
Note to myself: I am also expecting an update from setup-buildx-action@v1 to setup-buildx-action@v2

**Comment by johannbg** [comment]
Closing this one since I need more time to look at this.

**Pull request closed without merging**